README.md

```diff
@@ -1,158 +1,132 @@
-EMMO - Python API for the European Materials & Modelling Ontology
-=================================================================
+# EMMO - Python API for the European Materials & Modelling Ontology
```

Comment:
# EMMOntoPy - Python library for working with ontologies
EMMOntoPy is a Python library for woking with ontologies in general and the Elemental Multiperspective Material Ontology in particular.

Reply:
I think that the old header style with underline is more readable as text

Reply:
But that will mix styles, as the old style doesn't support headers at level 3 and on. So for header levels 1 and 2 it would be the underline styling and for sub-headers below that it would be the hash-styling. I find the mixing quite confusing.

If documents just have two header levels it's fine for me, but that then means that we would have mixed styles throughout our documentation.

Reply:
But in the end, if you feel strongly to keep the original styling in the markdown I guess this is fine - we should just do this all over then. That's the only thing I feel strongly about: Keeping the same style (whether it's mixed or not) all over the documentation.

Reply:
I've created an issue from this to continue the discussion: #231